baybe/searchspace/continuous.py

```diff
@@ -50,6 +58,15 @@ class SubspaceContinuous(SerialMixin):
     )
     """List of linear inequality constraints."""
 
+    constraints_cardinality: tuple[ContinuousCardinalityConstraint, ...] = field(
```

Comment:
didnt we agree to rename this into `nonlinear_consraints` or similar in accordance with the `optimize_*` singature?

or did we agree to do this in the PR that activates usage in `optimize_*` ?

Reply:
closing this thread since duplicate